packages/airnode-protocol/contracts/rrp/authorizers/admin/AirnodeFeeRegistryRolesWithManager.sol

```diff
@@ -0,0 +1,162 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.6;
+
+import "../../../access-control-registry/RoleDeriver.sol";
+import "../../../access-control-registry/AccessControlClient.sol";
+import "../../../access-control-registry/interfaces/IAccessControlRegistry.sol";
+import "./interfaces/IAirnodeFeeRegistryRolesWithManager.sol";
+
+/// @title Contract that implements generic AccessControlRegistry roles for
+/// the AirnodeTokenLock contract
+contract AirnodeFeeRegistryRolesWithManager is
+    RoleDeriver,
+    AccessControlClient,
+    IAirnodeFeeRegistryRolesWithManager
+{
+    // There are three roles implemented in this contract:
+    // Root
+    // └── (1) Admin (can grant and revoke the roles below)
+    //     ├── (2) Global Default Price setter
+    //     └── (3) Airnode Flag and Price setter
+    // Their IDs are derived from the descriptions below. Refer to
+    // AccessControlRegistry for more information.
```

Comment:
The reason why I didn't go for an owner was approach because it is highly likely that multiple addresses are going to be price setters i.e we don't want only 1 address to be able to control the price so we need to be able to have multiple owners which we could have done via a proxy to an only owner AirnodeFeeRegistry but then thats still a workaround. We might not need two roles but we definently need 1 role that can be assigned to muliple people.